src/libraries/System.Data.Common/src/System/Data/Common/DbException.cs

```diff
@@ -8,6 +8,8 @@ namespace System.Data.Common
     [System.Runtime.CompilerServices.TypeForwardedFrom("System.Data, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
     public abstract class DbException : System.Runtime.InteropServices.ExternalException
     {
+        public virtual bool IsTransient => false;
```

Comment:
Nit: We typically put properties after the constructors in the implementation.

Reply:
Hello @jkotas, do I need to submit a separate pull request to update the styling of the code file to match Microsoft expectations?

Reply:
You can push a new commit into this PR to fix this nit, no need to create a new PR.